src/test/java/com/application/bookingservice/repository/payment/PaymentRepositoryTest.java

```diff
@@ -0,0 +1,91 @@
+package com.application.bookingservice.repository.payment;
+
+import static com.application.bookingservice.model.Payment.Status;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.application.bookingservice.model.Booking;
+import com.application.bookingservice.model.Payment;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.jdbc.Sql;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class PaymentRepositoryTest {
+    private static Payment payment;
+    @Autowired
+    private PaymentRepository paymentRepository;
+
+    @BeforeAll
+    static void beforeAll() {
+        Booking booking = new Booking();

```

Comment:
create this field on the class level
```suggestion
        Booking booking = new Booking();

```